src/connectors/dynamic-widgets/connectDynamicWidgets.ts

```diff
@@ -156,7 +174,19 @@ const connectDynamicWidgets: DynamicWidgetsConnector =
 
           unmountFn();
         },
+        getWidgetSearchParameters(state) {
+          return state.setQueryParameters(additionalParameters);
+        },
         getRenderState(renderState, renderOptions) {
+          // todo: should this warn when additionalParameters.maxValuesPerFacet is undefined?
```

Comment:
should it warn when you explicitly leave out `maxValuesPerFacet` or is it worth warning about that behaviour (there's no point in on purpose requesting a lower number than what you're setting in limit)